This fixes the notab tests by converting them to spaces

--- scripts/opensuse-11.1/post-install.sh
+++ scripts/opensuse-11.1/post-install.sh
@@ -42,8 +42,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- scripts/opensuse-11.2/post-install.sh
+++ scripts/opensuse-11.2/post-install.sh
@@ -60,8 +60,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- scripts/opensuse-11.3/post-install.sh
+++ scripts/opensuse-11.3/post-install.sh
@@ -39,8 +39,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- scripts/opensuse-12.3/post-install.sh
+++ scripts/opensuse-12.3/post-install.sh
@@ -38,8 +38,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- scripts/opensuse-13.1/post-install.sh
+++ scripts/opensuse-13.1/post-install.sh
@@ -38,8 +38,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
